onnxruntime/core/session/provider_bridge_ort.cc

```diff
@@ -1235,6 +1235,7 @@ OrtCUDAProviderOptionsV2 OrtCUDAProviderOptionsToOrtCUDAProviderOptionsV2(const
   cuda_options_converted.has_user_compute_stream = legacy_cuda_options->has_user_compute_stream;
   cuda_options_converted.user_compute_stream = legacy_cuda_options->user_compute_stream;
   cuda_options_converted.default_memory_arena_cfg = legacy_cuda_options->default_memory_arena_cfg;
+  cuda_options_converted.enable_skip_layer_norm_strict_mode = legacy_cuda_options->enable_skip_layer_norm_strict_mode;
```

Comment:
Need not add to legacy cuda options. See the enable_cuda_graph in line 1241 below.